README.md

```diff
@@ -1,5 +1,10 @@
 # ![logo](https://quintilesims.github.io/layer0/static/logo_rs.png)
 
+[![Build Status](https://travis-ci.org/quintilesims/layer0.svg?branch=master)](https://travis-ci.org/quintilesims/layer0)  [![Version](https://img.shields.io/badge/version-0.9.0-brightgreen.svg)](https://github.com/quintilesims/layer0/releases/tag/v0.9.0)  [![Documentation](https://img.shields.io/badge/docs-latest-blue.svg)](https://quintilesims.github.io/layer0/)
```

Comment:
Huh, I don't know - it didn't occur to me to look into it. Do we have a pattern for templating `.md`s anywhere else in Layer0? I know we do some stuff with `L0_VERSION?=$(shell git describe --tags)` in our Makefiles.